jsonrpc-types/src/rpc_request/request.rs

```diff
@@ -112,6 +109,7 @@ macro_rules! define_call {
 
         #[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
         #[serde(untagged)]
+        #[allow(clippy::large_enum_variant)]   // TODO
```

Comment:
Need a real acceptable reason

Reply:
If we fix problem about `large_enum_variant` using `Box<>` in this pr . 
This pr must be unreadable.
Using Box<>, will influence all rpc reponse result, So, just use `#[allow]` temporary, i will fix it again in #224  after fix clippy issue.

Reply:
i'm not sure. TBD

you can continue to clippy the other crates.